src/Compilers/CSharp/Test/Emit/CSharpCompilerEmitTest.csproj

```diff
@@ -3,6 +3,7 @@
 <Project ToolsVersion="4.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <ImportGroup Label="Settings">
     <Import Project="..\..\..\..\..\build\Targets\VSL.Settings.targets" />
+	<Import Project="..\..\..\..\..\build\Targets\Roslyn.Toolsets.Xunit.targets" />
```

Comment:
Another slight nitpick...is there a reason we're Importing this at the top (rather than where it was at the bottom of most of the files)?  Having it at the top would allows for individual projects to easily override the .targets (that may not be something we want in this case).